src/main/java/com/backbase/ct/bbfuel/Runner.java

```diff
@@ -46,7 +46,10 @@ public void run(ApplicationArguments args) {
      * @throws IOException when setupAccessControl throws it
      */
     private void doIt() throws IOException {
-        LOGGER.info("Ingesting data into {}", GlobalProperties.getInstance().getString("environment.name"));
+        if (LOGGER.isInfoEnabled()) {
+            String environment = GlobalProperties.getInstance().getString("environment.name");
+            LOGGER.info("Ingesting data into {}", (environment == null ? "local environment" : environment));
```

Comment:
It does not have to be local ;-) it can also be remote

Reply:
environment is enough then